packages/utils/telemetry-utils/src/layerCompatError.ts

```diff
@@ -53,11 +57,12 @@ export function validateLayerCompatibility(
 				errorDetails: JSON.stringify(detailedProperties),
 			},
 		);
-		logger.send({
-			eventName: "LayerIncompatibilityError",
-			category: "error",
-			errorDetails: JSON.stringify({ ...coreProperties, ...detailedProperties }),
-		});
+		logger.sendErrorEvent(
+			{
+				eventName: "LayerIncompatibilityError",
+			},
+			error,
```

Comment:
Just to be totally clear - with this approach that I suggested, `coreProperties` will be top-level log event properties.  i.e. in our internal logging DB they'll likely land in the `Payload` thing at this point.  I could see an argument for putting it back the way it was so it's in the `errorDetails` bag.

Works either way, up to you!